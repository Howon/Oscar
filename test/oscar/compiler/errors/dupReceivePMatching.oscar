message testNow()


actor Tester() {
  
  receive = {
    | testNow() => {
        println("Error1");
	}
	| testNow() => {
	    println("Error2");
    }
  }
}

def main() => unit = {
  actor<Tester> tester1 = spawn actor<Tester>();
  message<testNow>() |> tester1;
}