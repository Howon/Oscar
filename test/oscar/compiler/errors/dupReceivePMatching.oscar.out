Error: Duplicate pattern matching in receive in actor Tester
